lib/internal/fs/promises.js

```diff
@@ -947,7 +968,7 @@ module.exports = {
     writeFile,
     appendFile,
     readFile,
-    watch,
+    watch: _watch,
```

Comment:
```suggestion
    watch: !isOSX && !isWindows ? _watch : watch,
```

Reply:
This is actually wrong. If it's Linux but does not have `recursive` option, it should call `watch`.

Reply:
It would call `watch` because of the `if` check in `_watch`, wouldn't it?

Reply:
That's also true. I guess, you are recommending this change as a optimization. 👍